spec/shipcloud/address_spec.rb

```diff
@@ -77,7 +81,7 @@
 
   def stub_addresses_request
     allow(Shipcloud).to receive(:request).
-      with(:get, 'addresses', {}).
+      with(:get, 'addresses', {}, api_key: nil).
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.